subjects/mobile-dev/stock-market/README.md

```diff
@@ -1,56 +1,35 @@
-# Stock Market
+## Stock Market
 
-### Exercise
+### Instructions
 
 Develop an app that will simulate a `real-time` stock market. You may use [Yahoo](https://algotrading101.com/learn/yahoo-finance-api-guide/#:~:text=Why%20should%20I%20use%20the%20Yahoo%20Finance%20API%3F,-Free&text=One%20good%20reason%20is%20because%20it%20can%20be%20completely%20free.) API for data retrieval.
+You should fetch all the data in real-time and choose 20 stocks to monitor. The objectives of this exercise are to practice fetching data in real-time, visualizing custom widgets in real-time, and implementing authentication and authorization services.
 
-Upon signing up, users will be given `1,000,000` fake dollars to use within the app for buying and holding stocks. The app should have the following features:
+Upon signing up, users must be given `1000000` fake dollars to use within the app for buying and holding stocks. The app should have the following features:
```

Comment:
Just a suggestion - I find it easier to read numbers with many digits when they're formatted. To avoid confusion with `,` or `.` I would use spaces: `1 000 000`